services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/events.py

```diff
@@ -250,6 +230,16 @@ async def action(cls, app: FastAPI, scheduler_data: SchedulerData) -> None:
                 dynamic_sidecar_endpoint
             )
         except BaseClientHTTPError:
+            were_service_containers_detected_before = (
+                len(scheduler_data.dynamic_sidecar.containers_inspect) > 0
+            )
+            if were_service_containers_detected_before:
+                # Containers disappeared after they were started.
+                # for now just mark as error and remove the sidecar
+                # NOTE: this is the correct place where to try and
+                # restart them (future use case).
+                raise
+
             # After the service creation it takes a bit of time for the container to start
             # If the same message appears in the log multiple times in a row (for the same
             # service) something might be wrong with the service.
```

Comment:
review that message. it means everything and nothing at the same time

Reply:
Renamed asking to investigate.

Reply:
hmm but you expect this to happen right after creation of the containers I believe. When the containers are not yet there. so this will also be confusing